course3/course_en.md

```diff
@@ -413,10 +400,10 @@ fn fib(n: Int) -> Int {
 
 ```moonbit
 fn even(n: Int) -> Bool {
-  n != 1 && (n == 0 || odd(n - 1))
+  n == 0 || odd(n - 1)
 }
 fn odd(n: Int) -> Bool {
-  n != 0 && (n == 1 || even(n - 1))
+  n == 1 || even(n - 1)
 }
 ```
```

Comment:
I simplified the implementation of `even` and `odd` to facilitate explanations in the text.